events/models.py

```diff
@@ -67,3 +61,29 @@ def save(self, *args, **kwargs):
 def validate_unique_user(event, user):
     if EventParticipant.objects.filter(event_id=event, user_id=user):
         raise ValidationError('user already exist in meeting')
+
+
+class PossibleMeeting(models.Model):
+    participant_id = models.ForeignKey(EventParticipant, on_delete=models.CASCADE)
+    date_time_start = models.DateTimeField()
+    date_time_end = models.DateTimeField()
+
+    def __str__(self) -> str:
+        return f"{self.participant_id} - {self.date_time_start}"
```

Comment:
Please add date_time_end like:
```suggestion
        return f"{self.participant_id} - {self.date_time_start} - {self.date_time_end}"
```
in this way we can see the difference if there are a few possible meetings with the same date_time_start but with different date_time_end

Reply:
Done